Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	1	clustering_fit
	1	clustering_transform
	1	fa
	1	fa_plot
	1	fa_sample_plot
	1	outliers
	7

rule outliers:
    input: data/fa.log
    output: data/outlier-removal.log
    jobid: 1

Finished job 1.
1 of 7 steps (14%) done

rule fa:
    input: data/single_cell_samples.tsv
    output: data/fa_likelihood.tsv, data/fa_factors.tsv, data/fa
    jobid: 3

Finished job 3.
2 of 7 steps (29%) done

rule clustering_fit:
    input: data/outlier-removal.log
    output: data/kmeans-fit.log
    jobid: 2

Finished job 2.
3 of 7 steps (43%) done

rule fa_sample_plot:
    input: data/fa-sample.log
    output: data/fa-sample-distribution.log
    jobid: 5

Finished job 5.
4 of 7 steps (57%) done

rule fa_plot:
    input: data/fa
    output: data/fa_likelihood-biplot.png
    jobid: 6

Error in rule fa_plot:
    jobid: 6
    output: data/fa_likelihood-biplot.png

RuleException:
CalledProcessError in line 40 of /Users/simondi/PROJECTS/target_infect_x_project/src/tix-analysis/biospark.snake:
Command ' set -euo pipefail;  1b-factor_analysis_plot.R -f data/fa ' returned non-zero exit status 127.
  File "/Users/simondi/PROJECTS/target_infect_x_project/src/tix-analysis/biospark.snake", line 40, in __rule_fa_plot
  File "/Users/simondi/anaconda3/envs/py36/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /Users/simondi/PROJECTS/target_infect_x_project/src/tix-analysis/.snakemake/log/2018-05-13T013758.648193.snakemake.log
